package.json

```diff
@@ -57,5 +57,8 @@
     ],
     "sourceMap": false,
     "instrument": false
+  },
+  "dependencies": {
+    "dag-map": "^2.0.2"
```

Comment:
This would need to go into babel-core since it's being used there. The top level package.json doesn't apply to any specific package and used for devDeps